docs/src/release_notes.rst

```diff
@@ -25,6 +25,10 @@ Release Notes
 
       - The UI now provides a formatted message for easily sharing onboarding details with validator operators.
 
+  - Validator
+
+       - Validator now supports a new scan-config, that allows to connect to a pre-specified set of SCAN apps. A threshold can be specified, which indicates how many of the SCAN apps need to agree on the result.
```

Comment:
I suggest we link to the docs related to this, which should contain info on how to set it up and more explanation about it.